src/main/java/chess/Piece.java

```diff
@@ -31,11 +32,37 @@ public void setSymbol(final Symbol symbol) {
     public void setStatus(final Status status) {
         this.status = status;
     }
-
-    public void setPosition(final Position position) {
-        this.position = position;
+    public void setPosition(final Position source) {
+        this.position = source;
+    }
+    public void setChessBoard(final ChessBoard chessBoard) {
+        this.chessBoard = chessBoard;
     }
 
+    public boolean move(final Position position) {
+        if (isPossibleMove(position)) {
+           if (chessBoard.getBoard()[this.position.getColumn()][this.position.getRow()] != null) {
+               chessBoard.getBoard()[this.position.getColumn()][this.position.getRow()].die();
+           }
+           //chess.getBoard()[this.position.getColumn()][this.position.getRow()] = null;
+            chessBoard.addPiece(this);
+           this.position = position;
+            chessBoard.getBoard()[this.position.getColumn()][this.position.getRow()] = this;
```

Comment:
indentation

Reply:
i made the changes could you review it please?